numpy/lib/_arraysetops_impl.py

```diff
@@ -350,6 +358,31 @@ def _unique1d(ar, return_index=False, return_inverse=False,
 
     optional_indices = return_index or return_inverse
 
+    if (optional_indices or return_counts) and not sorted:
+        raise ValueError(
+            "`sorted` can only be False if `return_index`, `return_inverse`, "
+            "and `return_counts` are all False."
```

Comment:
Should we add that we may add support in the future (e.g. for `return_counts`)?  Or just the fun word "currently" :)
(ValueError or TypeError seems reasonable to me, though)